spec.bs

```diff
@@ -3291,6 +3304,53 @@ at the expense of some utility.
   </wpt>
 </div>
 
+<h3 id=credentialless-monkeypatch>Iframe credentialless</h3>
+
+Note: The [[!IFRAME-CREDENTIALLESS]] specification defines a new object, the
+[=page credentialless nonce=]. At a high level, the
+[=fenced frame config instance/partition nonce=] serves the same purpose as the
+[=page credentialless nonce=] (partitioning storage and network), but scoped to each fenced
+frame rather than to each credentialless iframe. The
+[=fenced frame config instance/partition nonce=] is also used to revoke network access in fenced
+frames.
+
+Add the following algorithm:
+
+<div algorithm>
+  To <dfn for="browsing context">compute the effective partition nonce</dfn> given a
+  boolean |isCredentialless| and [=fenced frame config instance/partition nonce=] or null
+  |newFencedFrameNonce|:
+
+  1. If |credentialless| is true, return |this|'s [=top-level browsing context=]'s
```

Comment:
credentialless => isCredentialless

Reply:
I can't really change this, since it corresponds to an exposed IDL attribute.
https://wicg.github.io/anonymous-iframe/#spec-window-attribute

Reply:
I mean it's a local variable, but it would be a bit weird to call it something else when it corresponds exactly to this preexisting field, and the same name is used throughout the Iframe credentialless spec

Reply:
Where is this local variable defined? Sorry I don't think I'm getting it.

Reply:
Oh, I misread this. I didn't mean to name the variable `isCredentialless` above in the declaration. The iframe credentialless spec refers to this kind of variable as `credentialless` throughout, so it would be inconsistent naming to use `isCredentialless` here.

Reply:
Yes, I believe the "credentialless" variable needs to be declared. Reasoning:

- The "process a navigate fetch" algorithm below already has a "credentialless" variable 
- The "initialize a document object" algorithm below also already has a "credentialless" variable

Without it, we don't know whether the partition nonce should come from the credentialless iframe or the fenced frame.